contributing/editing-docs.rst

```diff
@@ -73,17 +73,15 @@ See :doc:`ci-docs`.
 Source
 ^^^^^^
 
-The Sphinx documentation is in the ``/docs/sphinx/`` folder in the code
-repository at `<https://github.com/openmicroscopy/bioformats>`_.
-
-Content for autogenerating documentation is in ``/components/autogen/src/``.
+The Sphinx documentation is decoupled from the code repository, at
```

Comment:
Is "decoupled from the code repository" something that needs explicitly mentioning?  It will only be of historical interest for anyone reading this.

Reply:
Yeah, but it's a change of the way it's been for years so seemed worth mentioning - only a few extra words anyway.